### PR TITLE
fix: added cli functionality to dataproc quickstart example

### PR DESCRIPTION
**Pull request by bradmiro**
The region tag is intentionally placed to include the CLI logic.

### CODE REVIEW

File: dataproc/quickstart/quickstart.py [hunk 1]
```diff
@@ -15,25 +15,24 @@
 # limitations under the License.
 
 # [START dataproc_quickstart]
+"""This quickstart sample walks a user through creating a Cloud Dataproc
+    cluster, submitting a PySpark job from Google Cloud Storage to the
+    cluster, reading the output of the job and deleting the cluster, all
+    using the Python client library.
+
+    Usage:
+        python3 quickstart.py --project_id <PROJECT_ID> --region <REGION> \
```
Comment (kurtisvg) [inline dataproc/quickstart/quickstart.py]: Is this a python3 only sample? I _think_ the official policy is we are still supporting 2.7 atm.

File: dataproc/quickstart/quickstart.py [hunk 2]
```diff
@@ -15,25 +15,24 @@
 # limitations under the License.
 
 # [START dataproc_quickstart]
+"""This quickstart sample walks a user through creating a Cloud Dataproc
```
Comment (kurtisvg) [inline dataproc/quickstart/quickstart.py]:
```suggestion
"""
This quickstart sample walks a user through creating a Cloud Dataproc
```

File: dataproc/quickstart/quickstart.py [hunk 3]
```diff
@@ -125,4 +124,23 @@ def quickstart(project_id, region, cluster_name, job_file_path):
     operation.result()
 
     print('Cluster {} successfully deleted.'.format(cluster_name))
-    # [END dataproc_quickstart]
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
```
Comment (kurtisvg) [inline dataproc/quickstart/quickstart.py]: This isn't tested at all - if we are going to add CLI functionality, we should have some tests verifying that it still works. 

Alternatively, I'd rather avoid CLI and instead give the user somewhere at the top of the code to set the variables instead.

File: dataproc/quickstart/quickstart.py [hunk 4]
```diff
@@ -125,4 +124,23 @@ def quickstart(project_id, region, cluster_name, job_file_path):
...
+        description=__doc__,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
+    parser.add_argument('--project_id', type=str,
```
Comment (kurtisvg) [inline dataproc/quickstart/quickstart.py]: nit: No hanging indents
```suggestion
    parser.add_argument(
      '--project_id', type=str,
```

File: dataproc/quickstart/quickstart.py (hunk 3 above)
Comment (bradmiro) [reply dataproc/quickstart/quickstart.py]: I can add a test for this, but the motivation for CLI is to make this a runnable tool as-is without needing to modify code.

**Pull request merged**